make/mt-config/mt-presets.mk

```diff
@@ -6,11 +6,12 @@
 include make/mt-config/mt-target-platform.mk
 
 # CONFIG presets
-CONFIG_POSIX_MAX                =bsp_posix-posix_fs-posix_platform-tls_bsp-senml-control_topic-memory_limiter
+CONFIG_POSIX_MAX                =bsp_posix-posix_fs-posix_platform-tls_bsp-control_topic-memory_limiter
 CONFIG_POSIX_MAX_THREADING      =bsp_posix-posix_fs-posix_platform-tls_bsp-senml-control_topic-threading-memory_limiter
 CONFIG_POSIX_MID                =bsp_posix-posix_fs-posix_platform-tls_bsp-senml-control_topic
 CONFIG_POSIX_MID_UNSECURE       =bsp_posix-posix_fs-posix_platform-senml-control_topic
 CONFIG_POSIX_MIN                =bsp_posix-posix_fs-posix_platform-tls_bsp
+CONFIG_POSIX_MIN_UNSECURE       =bsp_posix-posix_fs-posix_platform
```

Comment:
I think for not raising complexity more higher please remove the CONFIG_POSIX_MID_UNSECURE